app/routes/standardised-grant-amounts.js

```diff
@@ -40,7 +41,7 @@ module.exports = [{
       const result = await getStandardisedCosts(request.yar.id)
       console.log(result, '[THIS IS RESULT WE GOT BACK]')
       request.yar.set('standardisedCostObject', result)
-
+      await sendGAEvent({ name: eventTypes.STANDARDISED_COST, params: { standardised_cost: result }})
```

Comment:
this should be on landing on the /result-page page

Also, should be renamed as ELIGIBILITIES (as per madihas suggestion and confusion)